job.ci.build.yml

```diff
@@ -57,50 +57,60 @@ steps:
+    script: 'Import-Certificate -FilePath $(Agent.TempDirectory)\test-orch-sol-20-10.southindia.cloudapp.azure.com.cer -CertStoreLocation Cert:\LocalMachine\Root\'
 
-# The output of this task should not be published as it is only used for testing - Java 8 is still required for Jenkins, but it now also supports Java 11.
-# The reason for not using only Java 8 is that Sonar does not support Java 8 any longer.
-- task: Maven@3
-  displayName: 'Run unit tests (Java 11)'
+- task: DownloadSecureFile@1
+  displayName: "Download on-premise self signed orchestrator 21.10 certificate on build agent"
   inputs:
-    jdkVersionOption: 1.11
-    mavenPomFile: $(MavenPOMFile)
-    testResultsFiles: '**/TEST-*.xml'
-    sonarQubeRunAnalysis: true
-    findBugsRunAnalysis: true
-    sqMavenPluginVersionChoice: 'latest'
-    options: '-Djenkins.version=2.173 -Dsonar.projectKey=jenkins-plugin -Duser.name=UiPath --no-transfer-progress -Dmaven.artifact.threads=16 -T 4 -DrerunFailingTestsCount=0'
-  env:
-    TestOrchestratorUrl: $(TestOrchestratorUrl)
-    TestOrchestratorUsername: $(TestOrchestratorUsername)
-    TestOrchestratorPassword: $(TestOrchestratorPassword)
-    TestOrchestratorFolderName: $(TestOrchestratorFolderName)
-    
-    OrchestratorAuthorizationUriOverride: $(OrchestratorAuthorizationUriOverride)
-    OrchestratorClientIdOverride: $(OrchestratorClientIdOverride)
-
-    TestOrchestratorCloudUrl: $(TestOrchestratorCloudUrl)
-    TestOrchestratorCloudTenant: $(TestOrchestratorCloudTenant)
-    TestOrchestratorAccountName: $(TestOrchestratorAccountName)
-    TestOrchestratorAuthenticationToken: $(TestOrchestratorAuthenticationToken)
-    TestOrchestratorApplicationId: $(TestOrchestratorApplicationId)
-    TestOrchestratorApplicationSecret: $(TestOrchestratorApplicationSecret)
-    TestOrchestratorApplicationScope: $(TestOrchestratorApplicationScope)
-    TestOrchestratorCloudModernFolderName: $(TestOrchestratorCloudModernFolderName)
-    TestOrchestratorClientIdOverride: $(TestOrchestratorClientIdOverride)
-    TestOrchestratorAuthorizationUriOverride: $(TestOrchestratorAuthorizationUriOverride)
-    TestOrchestratorCloudUserName: $(TestOrchestratorCloudUserName)
-    TestOrchestratorCloudMachineName: $(TestOrchestratorCloudMachineName)
-    TestOrchestratorCloudUnattendedRobotName: $(TestOrchestratorCloudUnattendedRobotName)
-
+    secureFile: 'plugins-orch-21-10.westeurope.cloudapp.azure.com.cer'
 
+- task: PowerShell@2
+  displayName: "Install on-premise self signed orchestrator 21.10 certificate on build agent"
+  inputs:
+    targetType: 'inline'
+    script: 'Import-Certificate -FilePath $(Agent.TempDirectory)\plugins-orch-21-10.westeurope.cloudapp.azure.com.cer -CertStoreLocation Cert:\LocalMachine\Root\'
 
+# The output of this task should not be published as it is only used for testing - Java 8 is still required for Jenkins, but it now also supports Java 11.
```

Comment:
@ganeshborle Why have we commented this out ?

Reply:
I am also not sure as this change was made 4 months back, can we try uncommenting it and give a rerun?

Reply:
I strongly feel that I might have commented it to test something else.

Reply:
I just gave a run with this test task and found two issues now.

1. Previously the caffeine API was working correctly but now as it has new version it stopped working. `Caused by: java.io.IOException: Failed to load: Pipeline: Supporting APIs (workflow-support 838.va_3a_087b_4055b)
  Update required: Caffeine API Plugin (caffeine-api 2.9.2-29.v717aac953ff3) to be updated to 2.9.3-65.v6a_47d0f4d1fe or higher`
2. And other tests are failing because of the wrong username and password.
3. `System.NullReferenceException: Object reference not set to an instance of an object.`

Reply:
This has been taken care in this [PR ](https://github.com/jenkinsci/uipath-automation-package-plugin/pull/139) which is raised against this branch.